docs/manifest.json

```diff
@@ -129,10 +135,33 @@
       "path": "./secrets.md"
     },
     {
-      "title": "Users",
-      "description": "Learn about user roles available in Coder and how to create and manage users",
-      "icon_path": "./images/icons/users.svg",
-      "path": "./users.md"
+      "title": "Administration",
```

Comment:
Moved the configuration, auth, etc. from `install/` to `admin/". Also added "users" here.